installing/installing_gcp/preparing-to-install-on-gcp.adoc

```diff
@@ -0,0 +1,46 @@
+[id="preparing-to-install-on-gcp"]
+= Preparing to install on GCP
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-gcp
+
+toc::[]
+
+[id="preparing-to-install-on-gcp-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-gcp"]
+== Requirements for installing {product-title} on GCP
+
+Before installing {product-title} on Google Cloud Platform (GCP), you must configure a GCP project. See xref:../../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-project_installing-gcp-account[Configuring a GCP Project] for details about creating a project, enabling API services, configuring DNS, GCP account limits, and supported GCP regions.
+
+If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, see xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP] for other options.
+
+[id="choosing-an-method-to-install-ocp-on-gcp"]
+== Choosing a method to install {product-title} on GCP
+
+You can install {product-title} on GCP by using one of the following deployment methods.
+
+* **xref:../../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[Installing a cluster quickly on GCP]**: You can install {product-title} on GCP by using the default configuration options.
```

Comment:
Following [Michael's comment in this related PR](https://github.com/openshift/openshift-docs/pull/33447#discussion_r652101253), you could update this line to clarify that the infrastructure is installer-provisioned in this bullet point. For example:

"You can install OpenShift Container Platform on GCP infrastructure that is provisioned by the OpenShift Container Platform installation program. You can install a cluster quickly by using the default configuration options."